src/core/fonts.js

```diff
@@ -225,7 +225,8 @@ var ToUnicodeMap = (function ToUnicodeMapClosure() {
 
     forEach(callback) {
       for (var charCode in this._map) {
-        callback(charCode, this._map[charCode].charCodeAt(0));
+        if (typeof this._map[charCode].charCodeAt === 'function')
```

Comment:
The error is `This._map[charCode].charCodeAt is not a function`. When seeing that error, no fonts are loaded. When this line is added, the pdf renders perfectly. I thought dealing with the error directly would be better, but I can test if `typeof this._map[charCode] === 'string'` works.